modules/utils/notebook_utils/dataframe_helper.py

```diff
@@ -201,3 +201,37 @@ def generate_results_df(grid_results, curated_labels, pos_label=1):
             model_param_per[param] = [auc(recall, precision), auc(fpr, tpr)]
         performance_dict[lf_sample] = max(model_param_per.items(), key=operator.itemgetter(1))[1]
     return pd.DataFrame.from_dict(performance_dict, orient="index")
+
+def embed_word_to_index(cand, word_dict):
+    return [word_dict[word] if word in word_dict else 1 for word in cand]
+
+def generate_embedded_df(candidates, word_dict, max_length=83):
```

Comment:
What is this going to be used for, just curious?

Reply:
So this script update should have been in the next PR, but anyway this code will be used for generating word vectors and encoding sentences into a vector space the discriminator model will use to make predictions.